nitrite/src/main/java/org/dizitart/no2/mapper/MapperFacade.java

```diff
@@ -0,0 +1,19 @@
+package org.dizitart.no2.mapper;
```

Comment:
It has common methods with `NitriteMapper`. So let `MapperFacade` extends `NitriteMapper`. And let us add some documentations.